src/OmniSharp.MSBuild/ProjectManager.cs

```diff
@@ -178,6 +190,15 @@ private void ProcessQueue(CancellationToken cancellationToken)
                     foreach (var project in projectList)
                     {
                         UpdateProject(project.FilePath);
+
+                        // Fire loaded events
+                        if (project.LoadedEventArgs != null)
+                        {
+                            foreach (var eventSink in _eventSinks)
+                            {
+                                eventSink.ProjectLoaded(project.LoadedEventArgs);
```

Comment:
Actually, should you be try catching here? Or is that done at a higher layer?

Reply:
Yes, I should. I forgot to add that. I'll submit a new PR.